protocol/interoperable-ether-transfers.md

```diff
@@ -0,0 +1,261 @@
+# Purpose
+
+This document exists to align on a design for simplifying the process for sending `ETH` between two interoperable chains.
+
+# Summary
+
+New functionality is introduced to the `ETHLiquidity` contract that enables it to facilitate cross-chain `ETH` transfers to a specified recipient.
```

Comment:
I don't love adding new functionality to the `ETHLiquidity` contract as it has a ton of virtual ether and makes reasoning about it more difficult. Its previous design only allowed one possible `msg.sender`. I definitely prefer to put the logic in the bridge contract to keep the liquidity contract more simple

Reply:
+1 let's do this on the Bridge as originally planned :)

Reply:
update the proposed solution in the design doc to be adding this functionality to the `SuperchainWETH` contract.